catroid/src/org/catrobat/catroid/content/Script.java

```diff
@@ -59,8 +61,12 @@ public Script() {
 
 	public void doCopy(Sprite copySprite, Script cloneScript) {
 		ArrayList<Brick> cloneBrickList = cloneScript.getBrickList();
+		cloneScript.commentedOut = commentedOut;
 		for (Brick brick : getBrickList()) {
 			Brick copiedBrick = brick.copyBrickForSprite(copySprite);
+			if (!(copiedBrick instanceof ScriptBrick)) {
+				copiedBrick.setCommentedOut(brick.isCommentedOut());
+			}
```

Comment:
Because the comment out status of a script Brick is already set in the copy of the Script (line 64). The brick then is created using that script.

ScriptBricks store the status in the script, whereas other (Standard-)bricks store it internally, die to the code.xml structure